aardvark/manage.py

```diff
@@ -69,8 +56,13 @@ def run(self):
             if not ACCOUNT_QUEUE.empty():
                 (account_num, role_name, arns) = ACCOUNT_QUEUE.get()
```

Comment:
minor comment, should release lock once data is grabbed to ensure non-blocking ordering. I/O operations can take unbounded time

Reply:
Yeah, this is going to completely change with the retriever refactor. I'm planning on using asyncio and rewriting this logic.